### DISCUSSION TITLE
Why does NHiTS need the target variable specified in the time_varying_unknown_reals attribute?

### DISCUSSION THREAD

**Discussion by TeodorChiaburu** [jdb78/pytorch-forecasting#1683] [Q&A]
I was wondering why do I need to specify the target variable twice, when building a `TimeSeriesDataset` for a **NHiTS**? Once for the attribute 'target' and once for 'time_varying_unknown_reals'. If I don't specify it for the second attribute, I get a `ValueError: [target_name] is not in list.`

**Comment by KunzstBGR** [comment]
I discovered the same and would be interested in an answer. Is the N-HiTS architecture built in a way that the target has to be also an input (e.g. to predict rainfall I need historical rainfall values)? 
Other algorithms such as TFT can predict a target without having this variable as an input.

**Comment by benHeid** [comment]
I suppose this is more a discussion than an issue. Thus, I move it to discussion.
